pom.xml

```diff
@@ -796,9 +796,68 @@ under the License.</licenseText>
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.0</version>
+          <configuration>
+            <activeRecipes>
+              <!-- BestPractices -->
+              <!-- <recipe>org.openrewrite.java.recipes.JavaRecipeBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeNullabilityBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeTestingBestPractices</recipe> -->
+              <!-- testing -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.JUnitToAssertj</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.SimplifyAssertJAssertion</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.junit5.JUnit5BestPractices</recipe> -->
+              <!-- staticanalysis -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CodeCleanup</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CommonStaticAnalysis</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.FinalizeMethodArguments</recipe> -->
```

Comment:
`smell` never reassign params as we live in final land.

give dedication method and apply functional programming.

https://pmd.github.io/pmd/pmd_rules_java_bestpractices.html#avoidreassigningparameters

rewrite would make the boilerplate way i would prefer silent PMD.

https://adabeat.com/fp/immutability-in-functional-programming/

We make everywhere boilerplate to throw NPE ourselfs. This would really help to make thinks more robust. immutability is/should be the default case. Kotlin is all about being final and avoiding silly NPE. Thats why we want to live in final existence. Trying to reach invinity.